packs/spells/spellwrack.json

```diff
@@ -21,7 +21,7 @@
             "value": {}
         },
         "description": {
-            "value": "<p>You cause any spells cast on the target to spill out their energy in harmful surges. The target must attempt a Will save.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> Whenever the target becomes affected by a spell with a duration, the target takes @Damage[2d12[persistent,force]] damage. Each time it takes persistent force damage from spellwrack, it reduces the remaining duration of spells affecting it by 1 round. Only a successful Arcana check against your spell DC can help the target recover from the @UUID[Compendium.pf2e.conditionitems.Item.Persistent Damage]; the curse and the Persistent Damage end after 1 minute.</p>\n<p><strong>Failure</strong> As success, but the curse and Persistent Damage do not end on their own.</p>\n<p><strong>Critical Failure</strong> As failure, but the persistent force damage is @Damage[4d12[persistent,force]] damage.</p>"
+            "value": "<p>You cause any spells cast on the target to spill out their energy in harmful surges. The target must attempt a Will save.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> Whenever the target becomes affected by a spell with a duration, the target takes @Damage[2d12[persistent,force]] damage. Each time it takes persistent force damage from <em>spellwrack</em>, it reduces the remaining duration of spells affecting it by 1 round. Only a successful @Check[type:arcana|dc:resolve(@actor.attributes.spellDC.value)] check against your spell DC can help the target recover from the persistent damage; the curse and the persistent damage end after 1 minute.</p>\n<p><strong>Failure</strong> As success, but the curse and persistent damage do not end on their own.</p>\n<p><strong>Critical Failure</strong> As failure, but the persistent force damage is @Damage[4d12[persistent,force]].</p>"
```

Comment:
The issue here is multiple spellcasting entries. That will always use the highest spell DC.

Reply:
Fair enough. Removed the resolve.